module-types/Cisco/N7K-SUP2.yaml

```diff
@@ -0,0 +1,12 @@
+---
+manufacturer: Cisco
+model: N7K-SUP2
+part_number: N7K-SUP2
+comments: '[Cisco Nexus 7000 Switches Second-Generation Supervisor Modules Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/nexus-7000-series-switches/data_sheet_c78-710881.html)'
+interfaces:
+  - name: mgmt0
+    type: 1000base-t
+    mgmt_only: true
+console-ports:
+  - name: Con0
```

Comment:
This will throw a duplicate error if 2 sups are added. Not sure the best way to handle this IMO.

Reply:
I think we can prevent the duplicate error if we add the Sup{module} in front of Con0. 
console-ports:
name: Sup{module} Con0
type: rj-45